src/js/index.js

```diff
@@ -13,25 +14,41 @@ const makeShowUrl = async (id) => {
 
 // todo:-----------> Show them on HTML (name + image)
 
-const makeElementsForShow = async (Promisedata) => {
-  const data = await Promisedata;
-  const list = document.querySelector('#list');
-  const li = document.createElement('li');
-  li.className = 'item d-flex d-col center';
-  li.innerHTML = `
-  <img class="img" src="${data.image.original}" />
-  <div class="title d-flex center">
-    <h2 class="name">${data.name}</h2>
-    <a class="linkIcon"><i class="far fa-heart"></i></a>
-  </div>
-  <input type="button" value="Comments" />`;
-  list.appendChild(li);
+const makeElementsForShow = () => {
+  document.querySelector('#list').innerHTML
+    += '<li class="item"><img class="img"><div class="title d-flex center"><h2 class="name"></h2><a class="likes"><i class="far fa-heart"></i></a><p class="likesNum">0</p></div><input type="button" value="Comments"></li>';
```

Comment:
- [ ]  It may be better if you use ` template literal`  so that this line may be clear.